### ISSUE TITLE
[Feature Request] Prometheus Endpoint

### ISSUE THREAD

**Issue by 19wolf** [lizardfs/lizardfs#898]
I’d love to have an endpoint for Prometheus to scrape data from, such as chunks health, server usage, and other metrics available in the cgi-server.

**Comment by konstantin-921** [comment]
I know that the main resources are focused on developing the basic functionality of the project. But it would really be great to have a Prometheus endpoint to collect metrics.

**Comment by antonborecki** [comment]
@19wolf @konstantin-921 do you use LizardFS commercially and your businesses have a need for such a feature? Just curious about this need, maybe we'll put that on the roadmap :)

**Comment by konstantin-921** [comment]
Yes, we use LizardFS in commercial development. Right now there is a need to add monitoring for LizardFS. We use prometheus exporters to collect any metrics. It would be cool if you could add an endpoint for prometheus metrics.

**Comment by 19wolf** [comment]
I do not use LizardFS commercially, and in fact no longer use it at home either (for now), but if I were to use it at work I would definitely need this

**Comment by konstantin-921** [comment]
So far I have added "crutches". I'm running minute by minute requests via cron using lizardfs-admin. My cron job looks like this:

```
* * * * * sed -i 's/^chunks_availability .*/chunks_availability '$(/usr/bin/lizardfs-admin chunks-health <HOST> 9421 | awk 'NR==3{print $2}')'/g' /var/lib/node_exporter/textfile_collector/lizardfs.prom
```
Then I use Node Exporter Textfile Collector - https://github.com/prometheus/node_exporter#textfile-collector to handle this metrics. It's not the most convenient setup, but at least it works.

**Comment by antonborecki** [comment]
@konstantin-921 I've sent you an email about this. Please, check your inbox :)

**Comment by konstantin-921** [comment]
> @konstantin-921 I've sent you an email about this. Please, check your inbox :)

For some reason I don't see any new emails from you. Even the spam folder is empty. My email is konstantin-921@yandex.ru.

**Comment by antonborecki** [comment]
@konstantin-921 I've pinged you :) The e-mail address is correct. I am sending it from anton.borecki@lizardfs.com.
